### DISCUSSION TITLE
GitHub Copilot Extensions are now available in public preview for GitHub Mobile!

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#142240] [Mobile]
![Screenshot 2024-11-13 at 8 39 45 AM](https://github.com/user-attachments/assets/9f7fce7b-c79b-46fb-af86-236b7954ba82)

#### 🚀 **Heads up, devs—GitHub Copilot just leveled up!** 

GitHub Copilot Extensions are now available in public preview for GitHub Mobile!

With Copilot Extensions, you can extend GitHub Copilot’s capabilities on the go with GitHub Mobile. Use extensions to query third-party tools or private data in natural language.

How to join the preview program:

• On iOS: Install TestFlight and follow the prompts to join the GitHub Mobile beta program.  
• On Android: Become a beta tester on Google Play and follow the prompts to download the beta version of GitHub Mobile.

You can also build your own private extensions for internal use or publish extensions to the GitHub Marketplace. For more info, see our [docs on building Copilot Extensions](https://docs.github.com/en/copilot/building-copilot-extensions/about-building-copilot-extensions).

> [!IMPORTANT]
We have temporarily rolled back support for GitHub Copilot Extensions on JetBrains IDEs to perform maintenance. Extensions are not currently supported on JetBrains, but we’re working to restore access as soon as possible. Further updates will be posted to the GitHub Changelog. 

⭐ Interested in learning more about what you can do with extensions? Check out [Accelerate with AI: Build your own GitHub Copilot Extension](https://github.com/orgs/community/discussions/142312) a speaker talk from Universe 2024.

Already playing around with it? **Drop your thoughts below**—how are Copilot Extensions supercharging your dev flow? Let’s get creative! What integrations or features do you want to see next? 👇

[Read more on Docs
](https://docs.github.com/en/copilot/building-copilot-extensions/about-building-copilot-extensions)

**Comment by Akash1134** [comment]
🚀

**Comment by AndrewNikolin** [comment]
Has anyone got this working in any JB IDE? Tried both stable and nightly, and restarted IDE, yet the `@` shortcut doesn't seem to trigger any of the installed extensions. Autocomplete for the extension name doesn't show and the response (if I type in the name manually) doesn't look like coming from the extension. Works in web and GitHub Mobile on iOS though under same account

**Reply by transformsen** [reply]
Same problem. Using recent version of GH copilot in intelij but @ not working

@Akash1134 

Is there a sample/api details available for creating private extension for Jetbrains

**Reply by AndrewNikolin** [reply]
Feels like github doesn't care about JB extension at all. 3 stable releases and 7 nightly ones, since this announcement was written, yet functionality released here doesn't work. And in general, the extension isn't on par functionality-wise with extensions for other editors

**Comment by rnavaneeth090902** [comment]
Hi @Akash1134 

We are unable to trigger @ in Jetbrains

When will we get this update

Thanks

**Comment by D1M1TR10S** [comment]
Hi @AndrewNikolin @transformsen @rnavaneeth090902.

Apologies for the delayed access. We had to rollback the JetBrains release for maintenance. A fix is currently in progress.

I'll make sure to update this post in the meantime and once JetBrains support is available.

**Reply by D1M1TR10S** [reply]
Sharing an update here: we've applied the necessary fixes and re-released Copilot Extensions support for JetBrains IDE. It's now accessible if you want to give it a try again!

Here's the [new Community Discussion thread](https://github.com/orgs/community/discussions/146076) for sharing feedback and questions, and the [Changelog post](https://github.blog/changelog/2024-12-02-github-copilot-extensions-now-supported-in-jetbrains-ides/).

**Reply by D1M1TR10S** [reply]
@AndrewNikolin @transformsen @rnavaneeth090902

**Reply by transformsen** [reply]
@D1M1TR10S  Thanks a lot ❤.. It is working nicely. 
Is there any documents available for creating custom/private extension?

**Comment by Kishor074** [comment]
,, ,,

**Comment by Kishor074** [comment]
=
